src/sign.builds

```diff
@@ -36,7 +36,6 @@
   </Target>
 
   <Target Name="Build"
-          Condition="'$(SkipSigning)' != 'true' and '$(SignType)' != 'public'"
```

Comment:
What was the motivation for getting rid of these conditions?

Reply:
Those conditions are moved up to the outer `sign.proj` file.  The reason they are removed in the inner `sign.builds` file is so I didn't need to flow them from one MSBuild to the full fx MSBuild.  If `sign.proj` decides to call `sign.builds`, `sign.builds` should just do its job.

Reply:
Although, thinking about it a little more - does `$(SignType)` need to flow into this project?

Reply:
It may need to be there when we invoke `SignFiles` in sign.builds

Reply:
Ok, flowed SignType through to sign.builds.  Thanks for the catch.